frontend/src/AlertsManagement/management_helpers.test.js

```diff
@@ -0,0 +1,71 @@
+import { getAlertsData } from "./management_helpers";
+import { enableFetchMocks } from 'jest-fetch-mock';
+enableFetchMocks();
+
+describe("fetch alerts", () => {
+  // Fake alert JSON data. TODO: add ID property in the future.
+  const fakeAlerts = [{
+    anomalies: [
+      { dataPoints: {"2019-11-24": {value: 79}, }, 
+        dimensionName: "Ramen", metricName: "Interest Over Time",
+        timestampDate: {date: {year: 2019, month: 12, day: 29}}
+      },
+      { dataPoints: {"2019-10-27": {value: 53}, }, 
+        dimensionName: "Ramen", metricName: "Interest Over Time",
+        timestampDate: {date: {year: 2019, month: 12, day: 1}}
+      },
+    ],
+    status: "UNRESOLVED",
+    timestampDate: {date: {year: 2019, month: 12, day: 8}},
+  }, 
+  {
+    anomalies: [
+      { dataPoints: {"2019-10-24": {value: 46}, }, 
+        dimensionName: "Ramen", metricName: "Interest Over Time",
+        timestampDate: {date: {year: 2019, month: 11, day: 27}}
+      },
+    ],
+    status: "RESOLVED",
+    timestampDate: {date: {year: 2019, month: 12, day: 27}},
+  }];
+
+  // Expected allAlerts, unchecked, and checked.
+  const expectedAlerts = new Map();
+  expectedAlerts.set(0, {
+    timestampDate: "Sun Dec 08 2019",
```

Comment:
it seems that the date here has a day of the week, and the `fakeAlerts` date does not have day of the week

when you process the dates, do you append the day of the week for the `fakeAlerts`?

Reply:
Yeah it comes from the JavaScript Date